src/Php82/Php82.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Polyfill\Php82;
+
+/**
+ * @author Tim Düsterhus <duesterhus@woltlab.com>
+ *
+ * @internal
+ */
+final class Php82
```

Comment:
Do we need this class?

Reply:
See: https://github.com/symfony/polyfill/pull/404#discussion_r848428986